responsive-test-utils/src/test/java/dev/responsive/kafka/api/ResponsiveTopologyTestDriverKeyValueStoreTest.java

```diff
@@ -38,50 +42,56 @@
 import org.apache.kafka.streams.TopologyTestDriver;
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.ValueAndTimestamp;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 
 public class ResponsiveTopologyTestDriverKeyValueStoreTest {
 
+  private static final Instant STARTING_TIME = Instant.EPOCH;
+  private static final String STORE_NAME = "people";
+
   private static ResponsiveKeyValueParams paramsForType(final KVSchema type) {
     return type == KVSchema.KEY_VALUE
-        ? ResponsiveKeyValueParams.keyValue("people")
-        : ResponsiveKeyValueParams.fact("people");
+        ? ResponsiveKeyValueParams.keyValue(STORE_NAME)
+        : ResponsiveKeyValueParams.fact(STORE_NAME);
   }
 
   @ParameterizedTest
   @EnumSource(SchemaTypes.KVSchema.class)
   public void shouldRunWithoutResponsiveConnectionAndNoTtl(final KVSchema type) {
     // Given:
-    final TopologyTestDriver driver = setupDriver(paramsForType(type));
-
-    final TestInputTopic<String, String> bids = driver.createInputTopic(
-        "bids", new StringSerializer(), new StringSerializer());
-    final TestInputTopic<String, String> people = driver.createInputTopic(
-        "people", new StringSerializer(), new StringSerializer());
-    final TestOutputTopic<String, String> output = driver.createOutputTopic(
-        "output", new StringDeserializer(), new StringDeserializer());
-
-    // When:
-    people.pipeInput("1", "1,alice,CA");
-    people.pipeInput("2", "2,bob,OR");
-    people.pipeInput("3", "3,carol,CA");
-
-    bids.pipeInput("a", "a,100,1");
-    bids.pipeInput("b", "b,101,2");
-    bids.pipeInput("c", "c,102,1");
-    bids.pipeInput("d", "d,103,3");
-
-    // Then:
-    final List<String> outputs = output.readValuesToList();
-    MatcherAssert.assertThat(outputs, Matchers.contains(
-        "a,100,1,1,alice,CA",
-        "c,102,1,1,alice,CA",
-        "d,103,3,3,carol,CA"
-    ));
-    driver.close();
+    final Topology topology = topology(paramsForType(type));
+    try (final TopologyTestDriver driver = setupDriver(topology)) {
```

Comment:
no changes to any of the existing tests in this file, just moved the driver into a try-with-resources